### ISSUE TITLE
Meterpreter Session closed Reason Died...??

### ISSUE THREAD

**Issue by umair7917** [rapid7/metasploit-framework#6846]
Hey guys...!!!

I have an question? i m trying to hack a windows pc on WAN by creat a payload with msfvenom in kali linux 2016. i used my externl ip address during creat a payload and use a forward port 443 . i use metasploit v 4.11.23 in my kali linux machaine. when i opened the backdoor in my other pc which is connected to sa diffrent internet . matasploit takes very long dutration round about 5 mins to creat a meterpreter session.when i run a command sysinfo in meterpreter session it said unknow command. and then the error comes up that meterperter session will closed reason died..?? when i m trying to hack pc on same network then it works for me but for WAN it did'nt work for me.

is there any problem with my router configuration or in firewall setting??

Please Help me ...to solve this problem..    I will be very thankfull to you people..
![screenshot from 2016-05-03 17-46-07](https://cloud.githubusercontent.com/assets/19171876/14989052/0429e048-1124-11e6-8eb8-5313b3c703d5.png)
![screenshot from 2016-05-03 17-46-22](https://cloud.githubusercontent.com/assets/19171876/14989065/1530f8b8-1124-11e6-89ae-c262972357c5.png)

**Comment by bcook-r7** [comment]
LHOST must be routable from your remote host. I don't think that 192.168.8.100 is routable from 119.63.142.8. You need to make LHOST your internet-facing address.

**Comment by OJ** [comment]
Yup, LHOST is killing you as Brent pointed out. Have a read of the wiki,
this is covered in depth.

**Comment by umair7917** [comment]
how can i change it?? any idea??

**Comment by void-in** [comment]
@umair7917 Generate another payload through msfvenom and use the correct LHOST.

**Comment by join-us** [comment]
Any Anti-Virus software in your ```LHOST``` ?
